homework/shelkovich_ilya/circle/src/main/java/com/tutrit/java/quickstart/service/CircleService.java

```diff
@@ -0,0 +1,8 @@
+package com.tutrit.java.quickstart.service;
+
+public class CircleService {
+
+    public double calculateArea(double r){
+        return Math.PI * (r * r);
+    }
```

Comment:
As you named it **Circle**Service, it would be reasonable to have **Circle** in parameters instead of double r.
Please, try it that way.

Reply:
did it